res/TemplateBatchFiles/SelectionAnalyses/MEME.bf

```diff
@@ -1,4 +1,5 @@
 RequireVersion("2.3");
+OPTIMIZATION_TIME_HARD_LIMIT=1;
```

Comment:
Yes, when I noticed I kept the declaration in, I marked the pull request as `In Progress`. Thank you for catching it as well.